File: lua/refactoring/tests/refactor/106/go/struct-2/extract.expected.go
package main

import "fmt"

type foobar struct {
   poggers int
}

func (b *foobar) foo(a int, test int, test_other INSERT_PARAM_TYPE) {
    for idx := test - 1; idx < test_other; idx++ {
        fmt.Println(b.poggers, idx, a)
    }
}

func (b *foobar) simple_function(a int) {
    var test int = 1
    test_other := 1
    b.foo(a, test, test_other)
}

File: lua/refactoring/tests/refactor/106/rb/simple-singleton-function/extract.expected.rb

def self.foo_bar(a, test, test_other)
  [test, test_other].each do |v|
    puts "#{a} #{v}"
  end
end

def self.simple_function(a)
  test = 1
  test_other = 11

  foo_bar(a, test, test_other)
end
